crates/primitives/transactions/src/lib.rs

```diff
@@ -139,3 +141,19 @@ impl From<&Transaction> for TxType {
         }
     }
 }
+
+/// Broadcasted declare contract transaction v0.
+#[derive(Debug, Eq, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct BroadcastedDeclareTransactionV0 {
+    /// The address of the account contract sending the declaration transaction
+    pub sender_address: FieldElement,
```

Comment:
Hey @tdelabro 

Here actually in bootstrapper declareV0 was called before account was initiated. So If I pass 0x1 here will this class be declared with account 0x1 ?

Reply:
nvm tested and working 🚀. Merging